This patch adds some quote marks to make pod2man happier.  
The passages in question come from standard dist units.  I don't think
it's worthwhile to patch the standard units just to silence pod2man.
Instead, I think pod2man ought to be fixed.

For now, we'll at least keep most people happy and patch up
Porting/Glossary.

    Andy Dougherty  July 9, 1998

--- Glossary.mc	Wed Jul 22 13:58:45 1998
+++ Glossary	Wed Jul 22 13:58:45 1998
@@ -1879,7 +1879,7 @@
 	make_set_make='#'		# If your make program handles this for you,
 	make_set_make="MAKE=$make"	# if it doesn't.
 	I used a comment character so that we can distinguish a
-	'set' value (from a previous config.sh or Configure -D option)
+	'set' value (from a previous config.sh or Configure '-D' option)
 	from an uncomputed value.
 
 mallocobj (mallocsrc.U):
@@ -1984,7 +1984,7 @@
 	whole thing is then lower-cased.
 
 n (n.U):
-	This variable contains the -n flag if that is what causes the echo
+	This variable contains the '-n' flag if that is what causes the echo
 	command to suppress newline.  Otherwise it is null.  Correct usage is
 	    $echo $n "prompt for a question: $c".
 
@@ -2222,9 +2222,9 @@
 	/bin/sh, though it's possible that some systems will have /bin/ksh,
 	/bin/pdksh, /bin/ash, /bin/bash, or even something such as
 	D:/bin/sh.exe.
-	This unit comes before Options.U, so you can't set sh with a -D
+	This unit comes before Options.U, so you can't set sh with a '-D'
 	option, though you can override this (and startsh)
-	with -O -Dsh=/bin/whatever -Dstartsh=whatever
+	with '-O -Dsh=/bin/whatever -Dstartsh=whatever'
 
 shar (Loc.U):
 	This variable is defined but not used by Configure.
@@ -2344,7 +2344,7 @@
 
 spitshell (spitshell.U):
 	This variable contains the command necessary to spit out a runnable
-	shell on this system.  It is either cat or a grep -v for # comments.
+	shell on this system.  It is either cat or a grep '-v' for # comments.
 
 split (models.U):
 	This variable contains a flag which will tell the C compiler and loader
